src/FieldUnicity.php

```diff
@@ -294,7 +294,7 @@ public static function dropdownFields($itemtype, $options = [])
        //Search option for this type
         if ($target = getItemForItemtype($itemtype)) {
            //Do not check unicity on fields in DB with theses types
-            $blacklisted_types = ['longtext', 'text'];
+            $blacklisted_types = ['longtext'];
```

Comment:
This kind of generic rule should not be used. There is a `getUnallowedFieldsForUnicity()` method to handle fields that may cause performances issues.